### PR TITLE
feat: adds optional sorting parameter to FilterQuery

### PR DESCRIPTION
**Pull request by justin-cechmanek**
This PR adds the ability to sort results by a hash field if the field name is passed as a parameter to filter queries.

### CODE REVIEW

File: redisvl/query/query.py [hunk 1]
```diff
@@ -136,6 +136,7 @@ def __init__(
         return_fields: Optional[List[str]] = None,
         num_results: int = 10,
         dialect: int = 2,
+        sort_by: Optional[str] = None,
```
Comment (tylerhutcherson) [inline redisvl/query/query.py:139]: let's also add this support to vector queries too

**Review by tylerhutcherson — CHANGES_REQUESTED** [review]
let's add to vector queries as well since you can sort the resulting set by any given field in the index

File: redisvl/query/query.py (hunk 1 above)
Comment (justin-cechmanek) [reply redisvl/query/query.py:139]: Added to Vector and Range queries also

Comment (tylerhutcherson) [reply redisvl/query/query.py:139]: thanks!

**Review by tylerhutcherson — APPROVED** [review]

**Pull request merged**